examples/basic_mysql_connection/index.js

```diff
@@ -8,13 +8,26 @@ var connection = mysql.createConnection({
 	user: config.user,
 	password: config.password,
 	port: config.port,
-    database: config.database
+    database: config.database 
 });
 
 connection.connect();
+let sql = ['SELECT COUNT(*) as todo-Items FROM todos',
+			'SELECT * FROM todos where done',
+			'SELECT * FROM todos where !done',
+			'SELECT * FROM todos ORDER BY Id DESC',
+			'SELECT * FROM todos ORDER BY Id DESC LIMIT 1',
+			'SELECT * FROM todos WHERE Name like "%databases%"',
+			'INSERT INTO todos (Name,Done) VALUES ("go climbing",0)',
```

Comment:
Your queries look good, but the homework asked you to `Write the necessary SQL statements to complete the functionality of your application`, and all of your queries use hard-coded values - not very useful for a todo application!

Also, you haven't used any kind of security measures to prevent SQL injection in any of your queries.